INSTALL.rst

```diff
@@ -34,7 +113,8 @@ We build assets using `npm`. Make sure you have installed it system wide.
 
 .. code-block:: console
 
-    (inspirehep)$ npm update && npm install -g node-sass clean-css requirejs uglify-js
+    (inspirehep)$ sudo npm update
+    (inspirehep)$ sudo npm install -g node-sass clean-css requirejs uglify-js
```

Comment:
@jacquerie: I've added a note and a link to that npm docs page, let me know if that's enough (I don't think I should copy the whole stuff, and well, I think they deserve a visit for their info ;) )